src/Chapter2.hs

```diff
@@ -620,7 +635,12 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [Int] -> [Int]
+takeEven l =  go [] l
+    where
+        go :: [int] -> [int] -> [int]
+        go acc [] = acc
+        go acc n = go (acc ++ (take 1 n)) (drop 2 n)
```

Comment:
Very creative! 🎨 

Note that you can implement this function with the simple pattern matching as well. It requires fewer function calls and maybe (just maybe) a bit more efficient (at least by memory).